wiki/Rules/Visual_Content_Considerations/en.md

```diff
@@ -1,117 +1,52 @@
-
-Swimsuits that are too small or geared to be overtly revealing are generally not allowed if they are in good faith, considered questionable by a reasonable quantity of users.
-
-### Mild to moderate violence
-
-Scenes involving active combat, fighting or any other physical activity that may involve the presence of minor wounds or injuries is considered acceptable so long as there is not an excessive presence of gore, bloodshed, or distressing physical impairment.
+## Important things to consider
 
-### Artistic nudity
+At a basic level, we try to stick to the general idea of the [PEGI 12 classification](https://pegi.info/what-do-the-labels-mean) where possible, with special considerations for our community's needs and tastes.
 
-Any depiction of classical artwork or the rendition of the human form where the portrayal of partial nudity is done in a strictly artistic and non-sexual context may be allowed upon review from the GMT.
+All pictures should be judged as they are, without factoring in any other cases before them. Decide based on what you see.
 
-However, said artistic nudity must possess some contextual relevance to the other content it is associated with in order for this exception to apply.
+Please also understand that we are a global, multicultural community with widely varying thoughts, opinions, and values on the topic of what is and isn't allowed.
 
-An example of proper application of this rule would be for content using the *Immersion* band cover from Pendulum, which features a naked man and woman pressing their hands together in a sidelong shot in the center of the piece. The icon is relatively small, understated, but discernible, and is considered allowed due to its non-sexual nature and contextual relevance to tracks from that album as a whole.
+When in doubt, aim to accommodate the collective values shared by the largest portion of the osu! community in general.
 
-### Romance, flirting or intimate affection
+## Criteria
 
-Depiction of romantic acts or common intimacy between characters in an established narrative context is acceptable so long as it does not tread upon the rules listed above.
+Images or visual elements containing any of the following are **not allowed**:
 
-This also includes "flirty" depictions which are - in good faith - considered as playful, not unreasonably excessive, and not in violation of any of the other inappropriate content markers.
+- **sexual content involving or targeted at minors**
+- **excessive violence, gore, dismemberment, decapitation, or maiming**
+- **significant sexual innuendo**
+- **sexual posturing or solicitation**
+- **erotic content or graphic displays of sexuality**
+- **examples of drug use, preparation, or identification with illicit drug subculture**
+- **depictions of deliberately inflammatory political, cultural, religious, or social content**
```

Comment:
I think it's excessively bold, may go without it, considering the sentence above

Reply:
gonna leave it bold since it needs to draw people's attention